### PR TITLE
Initial content for the SIG events docs

### PR DESCRIPTION
**Pull request by afrittoli**
Setup the README with the content from the workstream README, the
charter and some more.
Archive meeting minutes to meetings_2020.

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

### CODE REVIEW

**Comment by afrittoli** [comment]
@cdfoundation/sig-events

**Review by salaboy — APPROVED** [review]
+1 to merge!

File: README.md [hunk 1]
```diff
@@ -1,8 +1,112 @@
...
+- Andreas Grimmer, [@agrimmer](https://github.com/agrimmer), Dynatrace
+- Emil Bäckmark, [@e-backmark-ericsson](https://github.com/e-backmark-ericsson), Ericsson
+- Ramin Akhbari, ([@rakhbari](https://github.com/rakhbari)), eBay
+- Mattias Linnér, [@m-linner-ericsson](https://github.com/m-linner-ericsson), Ericsson
+- Andrea Frittoli, [@afrittoli](https://github.com/afrittoli), IBM
+- Mauricio Salatino [@salaboy](https://github.com/salaboy), [Camunda](https://camunda.com) / [LearnK8s](https://learnk8s.io)
+- Steve Taylor [@sbtaylor15](https://github.com/sbtaylor15), [DeployHub](https://www.deployhub.com) / [Ortelius OS](https://ortelius.io)
+- Tracy Ragan [@tracyragan](https://github.com/tracyragan), [DeployHub](https://www.deployhub.com) / [Ortelius OS](https://ortelius.io)
+- Erik Sternerson [@erkist](https://github.com/erkist), doWhile
+- Cameron Motevasselani ([@link108](https://github.com/link108)), Armory
```
Comment (link108) [inline README.md:66]:
```suggestion
- Cameron Motevasselani ([@link108](https://github.com/link108)), [Armory](https://www.armory.io/)
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,112 @@
...
+
+Chairs and the TOC Sponsor of the SIG are
+
+* Emil Bäckmark ([@e-backmark-ericsson](https://github.com/e-backmark-ericsson)), Ericsson - Co-Chair
+* Andrea Frittoli ([@afrittoli](https://github.com/afrittoli)), IBM - Co-chair
+* Isaac Mosquera ([@imosquera](https://github.com/imosquera)), Armory - TOC Sponsor
+
+## Meetings
+
+SIG Events meets every even week on Mondays at 16:00UTC. (*See your timezone [here](https://time.is/1600_in_UTC)*).
```
Comment (m-linner-ericsson) [inline README.md:106]: In the PR for the creation of the SIG we stated

> The SIG meets on the 1st and 3rd Monday of each month, at 4 PM CET.

Did we still want the 1st and 3rd Monday?

Comment (afrittoli) [reply README.md:106]: Nice catch, I will fix that!

**Review by m-linner-ericsson — DISMISSED** [review]

**Review by link108 — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,8 +1,112 @@
...
+
+The group is working on a standardized event protocol that caters for technology agnostic machine-to-machine communication in CI/CD systems. This specification will be published, reviewed and agreed upon between relevant Linux Foundation projects/members.
+
+The group aims to provide reference implementations such as event listeners and event senders on top of CloudEvents.
+
+## History
+
+This is group started as a work-stream within the CDF SIG Interoperability.
+
+The forming of the workstream was suggested on a [recent SIG Interoperability meeting]( https://github.com/cdfoundation/sig-interoperability/blob/master/docs/meetings.md#may-28-2020) and its first meeting was held on June 8th 2020.
```
Comment (e-backmark-ericsson) [inline README.md:43]:
```suggestion
The forming of the workstream was suggested on a [SIG Interoperability meeting](https://github.com/cdfoundation/sig-interoperability/blob/master/docs/meetings_2020.md#May-28-2020) and its first meeting was held on June 8th 2020.
```

**Review by e-backmark-ericsson — CHANGES_REQUESTED** [review]
Just a small correction needed in one section. Otherwise it looks good :)

**Review by e-backmark-ericsson — APPROVED** [review]

**Pull request merged**